ctapipe/io/containers.py

```diff
@@ -75,6 +75,10 @@ class RawCameraContainer(Container):
     """
     Storage of raw data from a single telescope
     """
+    event_number = Item(-1, "telescope event number")
+    num_channels = Item(-1, "number of channels per pixel")
```

Comment:
I re-attributed the variables to the proper containers and added a DigiCamRawCameraContainer class deriving from RawCameraContainer in io/containers to treat the camera dependent info. 
 @kosack In the zfits.py I simply overwrite the RawCameraContainer items of the map in the RawDataContainer tel attribute.Does that sounds fine?